bin/git-create-branch

```diff
@@ -27,13 +35,18 @@ fi
 test -z $BRANCH && echo "branch required." 1>&2 && exit 1
 if [[ -n $REMOTE ]]
 then
-	git ls-remote --exit-code $REMOTE &>/dev/null
+	git ls-remote --exit-code $REMOTE 1>/dev/null 2>/tmp/ls-remote-error
```

Comment:
The `ls-remote-error` file will be created even there is not output to the stderr. We should remove it in all the paths.